package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "node_modules/.bin/_mocha  $(find . -name '*.spec.js')",
-    "test:lint": "node_modules/eslint/bin/eslint.js ./  || exit 0",
+    "test:lint": "eslint .",
```

Comment:
Thanks for the contribution!

The exit 0 was there to prevent the ugly `npm-debug` message if it failed. If we want to run it as a test, it would have to stay removed. 

Can you update this to run eslint on just the `lib` folder. I don't think eslint errors in the test directory should cause the build to fail.